packages/create-fuels/src/cli.ts

```diff
@@ -174,8 +190,17 @@ export const runScaffoldCli = async ({
     });
   }
 
-  if (!programsToInclude.contract && !programsToInclude.predicate && !programsToInclude.script) {
-    throw new Error('You must include at least one Sway program.');
+  while (!programsToInclude.contract && !programsToInclude.predicate && !programsToInclude.script) {
+    log(chalk.red('You must include at least one Sway program.'));
+
+    // Exit the program if we are testing to prevent hanging
+    if (testMode) {
+      process.exit(1);
+    }
```

Comment:
We are throwing using `throw new Error()` everywhere else but using `process.exit(1)` here. Intentional?

Reply:
Good catch! Adjusted